src/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+## The 'herbstluftwm' executable ##
+
+add_executable(herbstluftwm main.cpp)
+install(TARGETS herbstluftwm DESTINATION ${BINDIR})
+
+# additional sources – core/architectural stuff
+target_sources(herbstluftwm PRIVATE
+	arglist.cpp	arglist.h
```

Comment:
I don't know which viewer setting (for how to visualize tabs) you need for this to *not* look broken, but in every other case, it's not great (including GitHub). I recommend spaces here, at least between the words. In fact, I don't see why a single space wouldn't be just fine here.